tests/suites/test_suite_pk.function

```diff
@@ -417,11 +417,14 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_RSA_C:MBEDTLS_HAVE_INT64 */
+/* BEGIN_CASE depends_on:MBEDTLS_RSA_C */
 void pk_rsa_overflow( )
 {
     mbedtls_pk_context pk;
-    size_t hash_len = (size_t)-1;
+    size_t hash_len = SIZE_MAX;
+
+    if( SIZE_MAX <= UINT_MAX )
+        return;
```

Comment:
@gilles-peskine-arm's comment seems to apply to this test, too: The calls below pass `NULL` arguments for `sig` and `sig_len` to `pk_verify_ext`, `pk_verify` and `pk_sign`.